### PR TITLE
Update hug.md

### PR DESCRIPTION
**Pull request by shreyash00007**
Well sir, I just created a folder for the images and uploaded one Hope you like it I can 
Do this for all commands.

### CODE REVIEW

File: docs/commands/fun/hug.md [hunk 1]
```diff
@@ -9,4 +9,6 @@ Hug another user in the server.
 
 ## Usage
 
- - `$hug [@user]`
\ No newline at end of file
+ - `$hug [@user]`
+ 
+ ![](https://github.com/shreyash00007/docs/blob/master/Img/hug.PNG)
```
Comment (tjrgg) [inline docs/commands/fun/hug.md:14]: This appears to link to an image in your fork that is not included in this PR.

Comment (tjrgg) [inline docs/commands/fun/hug.md:14]: This appears to link to an image in your fork that is not included in this PR.

**Review by tjrgg — CHANGES_REQUESTED** [review]
An image you've linked to comes from your fork and wasn't included in the PR.

**Comment by shreyash00007** [comment]
> An image you've linked to comes from your fork and wasn't included in the PR.

For that I need to give the images to you I will submit all the images by tomorrow.

**Comment by nsylke** [comment]
Closing due to stale PR

**Pull request closed without merging**